vscode-react/src/components/deployments/organisms/deploymentsTableBody.component.tsx

```diff
@@ -145,31 +147,31 @@ export const AKDeploymentTableBody = ({ deployments }: { deployments?: Deploymen
 	return (
 		deployments &&
 		deployments.map((deployment: Deployment, index: number) => (
-			<AKTableRow key={deployment.deploymentId} isSelected={selectedDeploymentId === deployment.deploymentId}>
-				<AKTableCell onClick={() => getSessionsByDeploymentId(deployment.deploymentId)} classes={["cursor-pointer"]}>
+			<Row key={deployment.deploymentId} isSelected={selectedDeploymentId === deployment.deploymentId}>
+				<Cell onClick={() => getSessionsByDeploymentId(deployment.deploymentId)} classes={["cursor-pointer"]}>
 					{getTimePassed(deployment.createdAt)}
-				</AKTableCell>
-				<AKTableCell onClick={() => getSessionsByDeploymentId(deployment.deploymentId)} classes={["cursor-pointer"]}>
+				</Cell>
+				<Cell onClick={() => getSessionsByDeploymentId(deployment.deploymentId)} classes={["cursor-pointer"]}>
 					<div className="flex justify-center">
-						<AKDeploymentState deploymentState={deployment.state} />
+						<DeploymentStateLabel deploymentState={deployment.state} />
 					</div>
-				</AKTableCell>
-				<AKTableCell onClick={() => getSessionsByDeploymentId(deployment.deploymentId)} classes={["cursor-pointer"]}>
+				</Cell>
+				<Cell onClick={() => getSessionsByDeploymentId(deployment.deploymentId)} classes={["cursor-pointer"]}>
 					{getSessionStateCount(deployment, SessionStateType.stopped)}
-				</AKTableCell>
-				<AKTableCell onClick={() => getSessionsByDeploymentId(deployment.deploymentId)} classes={["cursor-pointer"]}>
+				</Cell>
+				<Cell onClick={() => getSessionsByDeploymentId(deployment.deploymentId)} classes={["cursor-pointer"]}>
 					{getSessionStateCount(deployment, SessionStateType.running)}
-				</AKTableCell>
-				<AKTableCell onClick={() => getSessionsByDeploymentId(deployment.deploymentId)} classes={["cursor-pointer"]}>
+				</Cell>
+				<Cell onClick={() => getSessionsByDeploymentId(deployment.deploymentId)} classes={["cursor-pointer"]}>
 					{getSessionStateCount(deployment, SessionStateType.error)}
-				</AKTableCell>
-				<AKTableCell onClick={() => getSessionsByDeploymentId(deployment.deploymentId)} classes={["cursor-pointer"]}>
+				</Cell>
+				<Cell onClick={() => getSessionsByDeploymentId(deployment.deploymentId)} classes={["cursor-pointer"]}>
 					{getSessionStateCount(deployment, SessionStateType.completed)}
-				</AKTableCell>
-				<AKTableCell onClick={() => getSessionsByDeploymentId(deployment.deploymentId)} classes={["cursor-pointer"]}>
+				</Cell>
+				<Cell onClick={() => getSessionsByDeploymentId(deployment.deploymentId)} classes={["cursor-pointer"]}>
```

Comment:
try use one `onClick` in `<Row  onClick={() => getSessionsByDeploymentId(deployment.deploymentId)}>`

Reply:
refactored